west.yml

```diff
@@ -188,7 +188,7 @@ manifest:
       groups:
         - hal
     - name: hal_nordic
-      revision: d5c70305b2389641b0a166d0714775a1b13319a2
+      revision: pull/231/head
```

Comment:
This change should come from an MDK release integrated in a new release of nrfx.